migrations/Version20230316140730.php

```diff
@@ -29,7 +29,7 @@ public function up(Schema $schema): void
         $this->addSql('CREATE TABLE realm_helpdesk (id INT AUTO_INCREMENT NOT NULL, realm VARCHAR(127) NOT NULL, email_address VARCHAR(255) DEFAULT NULL, web VARCHAR(255) DEFAULT NULL, phone VARCHAR(20) DEFAULT NULL, lang VARCHAR(4) NOT NULL, name VARCHAR(50) NOT NULL, INDEX IDX_D1E34E45FA96DBDA (realm), PRIMARY KEY(id)) DEFAULT CHARACTER SET utf8mb4 COLLATE `utf8mb4_unicode_ci` ENGINE = InnoDB');
         $this->addSql('CREATE TABLE realm_vhost (http_host VARCHAR(127) CHARACTER SET utf8mb4 DEFAULT \'\' NOT NULL COLLATE `utf8mb4_unicode_ci`, realm VARCHAR(127) CHARACTER SET utf8mb4 DEFAULT \'\' NOT NULL COLLATE `utf8mb4_unicode_ci`, INDEX realm (realm), PRIMARY KEY(http_host)) DEFAULT CHARACTER SET utf8mb4 COLLATE `utf8mb4_unicode_ci` ENGINE = InnoDB COMMENT = \'\' ');
         $this->addSql('CREATE TABLE vhost_realm (id INT AUTO_INCREMENT NOT NULL, realm VARCHAR(127) NOT NULL, http_host VARCHAR(255) NOT NULL, path_prefix VARCHAR(255) NOT NULL, auth_service VARCHAR(80) NOT NULL, auth_config LONGTEXT DEFAULT NULL, INDEX IDX_C76ED703FA96DBDA (realm), PRIMARY KEY(id)) DEFAULT CHARACTER SET utf8mb4 COLLATE `utf8mb4_unicode_ci` ENGINE = InnoDB');
-        $this->addSql('CREATE TABLE contact (id INT AUTO_INCREMENT NOT NULL, name_id VARCHAR(150) CHARACTER SET utf8mb4 NOT NULL COLLATE `utf8mb4_unicode_ci`, display_name VARCHAR(255) CHARACTER SET utf8mb4 NOT NULL COLLATE `utf8mb4_unicode_ci`, email_address VARCHAR(255) CHARACTER SET utf8mb4 NOT NULL COLLATE `utf8mb4_unicode_ci`, password VARCHAR(255) CHARACTER SET utf8mb4 NOT NULL COLLATE `utf8mb4_unicode_ci`, super_admin TINYINT(1) DEFAULT 0 NOT NULL, PRIMARY KEY(id)) DEFAULT CHARACTER SET utf8mb4 COLLATE `utf8mb4_unicode_ci` ENGINE = InnoDB COMMENT = \'\' ');
```

Comment:
In general, it is wiser to create a new migration instead of modifying the migration to avoid problems with other systems using the same database.

Reply:
Right, I assumed nobody has this in production yet ;-) That's also why I'm inclined to modify this migration for issue #18 still, hope you can find some time to look at #18 actually